app/controllers/api/internal/helper/purchases_controller.rb

```diff
@@ -526,13 +528,24 @@ def reassign_purchases
         end
       end
 
-      count += 1 if purchase.save
+      if purchase.save
+        count += 1
```

Comment:
Do we still need `count`? Can we just use `reassigned_purchase_ids`? (actually, why don't we add the IDs to the response as well to have a bit more debugging information?)